lib/oneview-sdk/resource/api200/firmware_bundle.rb

```diff
@@ -22,7 +22,7 @@ class FirmwareBundle
       # @param [OneviewSDK::Client] client The client object for the OneView appliance
       # @param [String] file_path
       # @param [Integer] timeout The number of seconds to wait for completing the request
-      # @return [OneviewSDK::FirmwareDriver] if the upload was sucessful, return a FirmwareDriver object
+      # @return [OneviewSDK::FirmwareDriver] if the upload was successful, return a FirmwareDriver object
       def self.add(client, file_path, timeout = READ_TIMEOUT)
```

Comment:
@jsmartt , I thought the same, but I thought of opening a new problem for these changes and soon closing the scope of the i3s. But if you think better now, no problem for me.